code/game/world.dm

```diff
@@ -132,7 +141,7 @@ GLOBAL_PROTECT(security_mode)
 			break
 
 	if((!handler || initial(handler.log)) && config && CONFIG_GET(flag/log_world_topic))
-		WRITE_FILE(GLOB.world_game_log, "TOPIC: \"[T]\", from:[addr], master:[master], key:[key]")
+		log_world("TOPIC: \"[T]\", from:[addr], master:[master], key:[key]")

```

Comment:
This changed the log file?

Reply:
I'm pretty sure this was done on purpose because the log world procs don't work at this early stage iirc?

Reply:
Yeah, that was an oversight. The game log should be available, I'll add a proc for this case.